widgets/search-box.js

```diff
@@ -60,6 +70,12 @@ function searchbox(params) {
           input.value = state.query;
         }
       });
+
+      if (autofocus === 'true' ||
```

Comment:
does it means we are accepting people pasing autofocus: 'true'? I do not see why we would want that?